test/specs/behaviors/testDefinitions.ts

```diff
@@ -86,6 +87,81 @@ definitions.push({
   },
 })
 
+// Adds attribute 'aria-selected=true' to 'anchor' component's part based on the property 'active'. This can be overriden by directly providing 'aria-selected' property to the component.
+definitions.push({
+  regexp: /Adds attribute '([a-z A-Z -]+)=([a-z 0-9]+)' to '([a-z -]+)' component's part based on the property '([a-z]+)'\. This can be overriden by directly providing '([a-z -]+)' property to the component\./g,
```

Comment:
why we won't test for `This can be overriden by directly providing '([a-z -]+)' property to the component\.` expression only and introduce tests for verifying that only? Just imagine how each new sentence would complicate the overall picture if we would need to introduce new tests for each possible combination of conditions.